setup.py

```diff
@@ -71,7 +71,11 @@ def get_version(rel_path):
             "stack_output_external ="
             "sceptre.resolvers.stack_output:StackOutputExternal",
             "no_value = sceptre.resolvers.no_value:NoValue",
+            "select = sceptre.resolvers.select:Select",
             "stack_attr = sceptre.resolvers.stack_attr:StackAttr",
+            "sub = sceptre.resolvers.substitute:Sub",
```

Comment:
shouldn't this be `sub = sceptre.resolvers.sub:Sub`?

Reply:
Yikes! Yes! That's probably why my tests are failing.